app/services/ipam/base.rb

```diff
@@ -17,6 +17,18 @@ def initialize(opts = {})
       normalize_mac!
     end
 
+    def subnet_range
+      @subnet_range ||= begin
+        subnet_range = IPAddr.new("#{subnet.network}/#{subnet.mask}", subnet.family).to_range
+        # exclude first element - network
```

Comment:
Well, it can't hurt to skip these addresses also for IPv6. Generally speaking: Yes, IPv6 doesn't have a broadcast address.